package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fetchr",
-  "version": "0.5.24",
+  "version": "0.6.0",
```

Comment:
Ok just go to 0.5.25?
On Jan 27, 2016 6:36 PM, "Seth Bertalotto" <notifications@github.com> wrote:

> In package.json
> <https://github.com/yahoo/fetchr/pull/138#discussion_r51068670>:
>
> > @@ -1,6 +1,6 @@
> >  {
> >    "name": "fetchr",
> > -  "version": "0.5.24",
> > +  "version": "0.6.0",
>
> Can you revert this change? Don't think we will need to bump minor.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yahoo/fetchr/pull/138/files#r51068670>.
>